esp/src/src/react/components/Activities.tsx

```diff
@@ -0,0 +1,277 @@
+import { Gauge } from "@hpcc-js/chart";
+import { GetTargetClusterUsageEx, SMCActivity } from "@hpcc-js/comms";
+import { Avatar, Card, CardContent, CardHeader, createStyles, Divider, IconButton, makeStyles, Table, TableBody, TableCell, TableContainer, TableRow, MenuItem, Menu, CardActions, Collapse, Typography } from "@material-ui/core";
+import { green, red } from "@material-ui/core/colors";
+import Grid from "@material-ui/core/Grid";
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import clsx from "clsx";
+import * as React from "react";
+import { ActivityConnection } from "../../comms/activity";
+import { VisualizationComponent } from "./VizAdapter";
+
+
+import "dojo/i18n";
+// @ts-ignore
+import * as nlsHPCC from "dojo/i18n!hpcc/nls/hpcc";
+
+const useStyles = makeStyles(theme =>
+    createStyles({
+        root: {
+            // minWidth: 192,
+            // padding: 16
+            maxHeight: 320
+        },
+        green: {
+            color: theme.palette.getContrastText(green[500]),
+            backgroundColor: green[500],
+        },
+        red: {
+            color: theme.palette.getContrastText(red[500]),
+            backgroundColor: red[500],
+        },
+        bullet: {
+            // display: "inline-block",
+            // margin: "0 2px",
+            // transform: "scale(0.8)"
+        },
+        title: {
+            // fontSize: 14
+        },
+        pos: {
+            // marginBottom: 12
+        },
+        expand: {
+            transform: 'rotate(0deg)',
+            marginLeft: 'auto',
+            transition: theme.transitions.create('transform', {
```

Comment:
MInor: Mixed quotes.

Reply:
I should probably add linting for these...